baseplate/frameworks/thrift/__init__.py

```diff
@@ -82,7 +82,10 @@ def _set_remote_context(self, request_context: RequestContext) -> Iterator[None]
             header_dict = {}
             for k, v in headers.items():
                 try:
-                    header_dict[k.decode()] = v.decode()
+                    # this is only for w3c trace headers
+                    key = k.decode()
+                    if key == "traceparent" or key == "tracestate":
```

Comment:
Just checking, are we sure these are always lowercase? I would expect them to be since WSGI normalizes header capitalization but probably worth checking.

Reply:
They should only be being set by the otel lib at this point and the spec is lowercase. I think we should be good.

Reply:
But maybe we should just do a non case-sensitive test now that we have the old trace headers.